src/components/space/ActionsMap.js

```diff
@@ -111,6 +122,13 @@ function ActionsMap() {
                         20,
                       )}px`,
                     }}
+                    onClick={() => handleClusterZoom(longitude, latitude)}
+                    onKeyPress={(event) => {
+                      if (event.keyCode === 13)
```

Comment:
Avoid not wrapping `if` statements in `{}`, even if they are just one line.

Reply:
👍